pkg/virt-launcher/virtwrap/nichotplug_test.go

```diff
@@ -401,3 +469,17 @@ type fakeVMConfigurator struct {
 func (fvc *fakeVMConfigurator) SetupPodNetworkPhase2(*api.Domain, []v1.Network) error {
 	return fvc.expectedError
 }
+
+func vmiWithDefaultIfaceWithState(state v1.InterfaceState) *v1.VirtualMachineInstance {
+	vmi := libvmi.New(libvmi.WithInterface(*v1.DefaultBridgeNetworkInterface()))
+	vmi.Spec.Domain.Devices.Interfaces[0].State = state
```

Comment:
nit: The following commit introduces `libvmi.InterfaceWithState`, it could be useful here as well.